geoips/commandline/geoips_get.py

```diff
@@ -7,38 +7,170 @@
 
 import yaml
 
-from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+from geoips.commandline.geoips_command import (
+    CommandClassFactory,
+    GeoipsCommand,
+    GeoipsExecutableCommand,
+)
 from geoips.create_plugin_registries import format_docstring
 from geoips import interfaces
 
 
-class GeoipsGetFamily(GeoipsExecutableCommand):
-    """Get Command which retrieves and outputs information about a GeoIPS Family.
+class GeoipsGetInterface(GeoipsExecutableCommand):
+    """Get Command which retrieves information about a from or about a GeoIPS interface.
+
+    Where this artifact is one of ['interface', 'plugin', 'family'.]
+
+    This is called via `geoips get <interface_name> <opt_args>`. Data included when
```

Comment:
Yes, since that's largely just a nomenclature change, I'm waiting to implement that until we get all of the CLI PRs merged in. Don't want to deal with annoying conflicts for a non-functional change.

Reply:
Good plan ...